src/nxt_router.c

```diff
@@ -1959,12 +1959,22 @@ nxt_router_conf_create(nxt_task_t *task, nxt_router_temp_conf_t *tmcf,
         next = 0;
 
         for ( ;; ) {
+            int               listen_backlog = -1;
+            nxt_conf_value_t  *backlog;
+
+            static const nxt_str_t  backlog_path = nxt_string("backlog");
```

Comment:
Hi @ac000,
What about moving it into `nxt_router_listener_conf`?
For example:
```
diff --git a/src/nxt_router.c b/src/nxt_router.c
index 43209451..fe4bbabc 100644
--- a/src/nxt_router.c
+++ b/src/nxt_router.c
@@ -1964,13 +1964,10 @@ nxt_router_conf_create(nxt_task_t *task, nxt_router_temp_conf_t *tmcf,
                 break;
             }

-            skcf = nxt_router_socket_conf(task, tmcf, &name);
-            if (skcf == NULL) {
-                goto fail;
-            }
-
             nxt_memzero(&lscf, sizeof(lscf));

+            lscf.backlog = -1;
+
             ret = nxt_conf_map_object(mp, listener, nxt_router_listener_conf,
                                       nxt_nitems(nxt_router_listener_conf),
                                       &lscf);
@@ -1981,6 +1978,13 @@ nxt_router_conf_create(nxt_task_t *task, nxt_router_temp_conf_t *tmcf,

             nxt_debug(task, "application: %V", &lscf.applicatio
n);

+            /* use lscf.backlog */
+
+            skcf = nxt_router_socket_conf(task, tmcf, &name);
+            if (skcf == NULL) {
+                goto fail;
+            }
+
             // STUB, default values if http block is not define
d.
```
I usually prefer to get object property by `nxt_conf_map_object()` rather than explicitly `nxt_conf_get_object_member()`.

Others look good.

Reply:
I did try that originally but then it started getting messy... let me take another look...

Reply:
OK, so it means

1) Adding a new 'backlog' entry to nxt_router_listener_conf[]

2) Adding a new 'backlog' member to nxt_router_listener_conf_t

2) Adding a pointer to nxt_router_listener_conf_t in nxt_router_temp_conf_t

So I took the simpler route, but sure, I can do the above instead...